.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug report
```

Comment:
There's now a bug_report.yml and bug_report.yaml

Reply:
sorry about that. Sorted it now